templates/events-list.php

```diff
@@ -23,9 +35,12 @@
 		<?php
 		while ( $query->have_posts() ) :
 			$query->the_post();
+			$event_start = ( new DateTime( get_post_meta( get_the_ID(), '_event_start', true ) ) )->format('l, F j, Y');
 			?>
-			<li>
+			<li class="event-list-item">
+				<span class="event-list-date"><?php echo esc_html( $event_start ); ?></span>
 				<a href="<?php the_permalink(); ?>"><?php the_title(); ?></a> by <span><?php the_author(); ?></span>
```

Comment:
When testing this on wp-env, `the_permalink()` sends me to `/events/test-2/` which 404s. Should it instead send to `/glotpress/events/test-2/`?

Reply:
Yea, nice catch. Thank you!